src/main/webapp/user_settings.html

```diff
@@ -36,7 +36,7 @@ <h3>Change Skills:</h3>
           </ul>
         </form>
         <h3>Change Other Information:</h3>
-        <form id="stats-form">
+        <form id="stats-form" action="/create-user" method="POST">
```

Comment:
Try to build all of these in a single form.  Each form would be 1 submit to the servlet, and you'd want the submit to have complete information to be able to create the user